lib/src/presentation/view/home/widgets/service_category_list.dart

```diff
@@ -1,8 +1,6 @@
 
 import 'package:coofix/app/router/router_constants.dart';
 import 'package:coofix/src/application/get_servieces/get_services_bloc.dart';
-import 'package:coofix/src/application/get_servieces/get_services_event.dart';
-import 'package:coofix/src/application/get_servieces/get_services_state.dart';
 import 'package:coofix/src/presentation/core/constants/constants.dart';
 import 'package:coofix/src/presentation/core/constants/images.dart';
 import 'package:coofix/src/presentation/core/theme/colors.dart';
```

Comment:
onTap: () {
                       
    context.read<GetServicesBloc>().add(
        const GetServicesEvent.getServices(
            limit: 0, skip: 0, id: ""),
        );
    Navigator.pushNamed(
        context, RouterConstants.serviceDetailRoute,
    );
},

Why calling getServices event while click on a service

Please ensure the placemeent services logic is working properly. Because the count of services are hardcoded. Please make sure its is working properly